start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js

```diff
@@ -1,150 +1,162 @@
-windowObjectWrap.NewEvaluationPage = function(options) {
-    this.render = `
-    ${windowObjectWrap.NAV()}
-    <section class="form-candidate">
-        <div class="container">
-            <div class="row">
-                <div class="col-md-12">
-                    <form action="" class="user-input" id='form-new' method='post'>
-                        ${Inputs(
-                            windowObjectWrap.InputData()
-                        )}
-                        ${TechnicalLevelPicker(
-                            windowObjectWrap.TechnicalData()
-                        )}
-                        ${TextArea(
-                            windowObjectWrap.TextareaData()
-                        )}
-                        ${SelectOption(
-                            windowObjectWrap.FieldsetData()
-                        )}
-                        <div class="button-wrap">
-                            <input type="button" id='btn' class='submit-button' value='Submit'>
+var xhr = new XMLHttpRequest();
+xhr.open('GET','../../data/data.js',true);
+xhr.onload = function(){
+    if(this.status == 200){
+        windowObjectWrap.NewEvaluationPage = function(options) {
```

Comment:
Any particular reason why this is deferred until the HTTP request is loaded?

Reply:
No reason at all. I was just experimenting. I've just discovered now the differences between onload and onreadystatechange -> https://stackoverflow.com/a/19247992/7095201